addons/web/static/src/xml/web_calendar.xml

```diff
@@ -77,7 +77,7 @@
                             </span>
                             <i t-if="filter.value == 'all'" class="o_cw_filter_avatar fa fa-users fa-fw  flex-shrink-0 mr-1" role="img" aria-label="Avatar" title="Avatar"/>
                             <img t-elif="widget.avatar_field and filter.value" t-attf-src="/web/image/#{widget.avatar_model}/#{filter.value}/#{widget.avatar_field}" class="o_cw_filter_avatar flex-shrink-0 mr-1" alt="Avatar"/>
-                            <span class="o_cw_filter_title text-truncate flex-grow" t-esc="filter.label" t-att-title="filter.label"/>
+                            <span class="o_cw_filter_title text-truncate flex-grow" t-esc="filter.label" t-attf-title="#{ ['all', false].includes(filter.value) or !widget.avatar_field ? filter.label : '' }"/>
```

Comment:
There are some duplication of logic here.
From what I see, this happens because popover is configured by calendar_renderer just after appending the filter, and this template here is the one of the filter itself.
I don't see why the making of popover is not made by the filter... If that was the case, then it would be much easier to see that it makes either the popover or the title.

Reply:
@alexkuhn There is a task in which the **popover code** has been moved to the **SidebarFilter**: https://github.com/odoo/odoo/pull/42455/files#diff-f33bfd31745ec1ccc8a727020e0a3a57

So should we move the popover code into the SidebarFilter in this task?
Or should we add an else condition to overcome the duplication of logic?

Reply:
All things considered, I'm good with code duplication with this PR.
OWL version will not have code duplication issue, and I don't think it's worth making more changes than necessary with this PR.